leader/leader.go

```diff
@@ -196,9 +197,14 @@ func (l *Client) AddWatch(key string, retry time.Duration, valuesC chan string)
 					continue
 				}
 			}
-
+			// if nothing has changed and we previously sent this subscriber values,
+			// do not bother subscriber with extra notifications
+			if re.PrevNode != nil && re.PrevNode.Value == re.Node.Value && sentAnything {
+				continue
+			}
 			select {
 			case valuesC <- re.Node.Value:
+				sentAnything = true
```

Comment:
This looks like a one-time thing and is not useful as such - shouldn't you reset it once you've received an actual update (prevValue != value)?
Maybe I misunderstood the intention - is this to cover the case of protecting against sending  duplicate updates once the first update has been sent?